exonum/src/node/mod.rs

```diff
@@ -514,11 +482,6 @@ impl NodeHandler {
         let is_enabled = api_state.is_enabled();
         api_state.set_node_role(node_role);
 
-        let config_manager = match config_file_path {
```

Comment:
Nit / slightly off-topic: Could you please fix the usage of `NodeHandler.config_manager` in `node/events.rs`?

```rust
if self.config_manager.is_some() {
    // ...
    self.config_manager.as_ref().unwrap().store_connect_list(/* ... */)
}
```

can be replaced with `if let` construction.